giphy.js

```diff
@@ -0,0 +1,57 @@
+$(document).ready(function(){
+
+  $("button").on("click", function () {
+    var animals = $(this).attr("data-animals");
+    var queryURL = "https://api.giphy.com/v1/gifs/search?q=" +
+      animals + "&api_key=dc6zaTOxFJmzC&limit=10";
+  
+    $.ajax({
+      url: queryURL,
+      method: "GET"
+    })
+      .then(function (response) {
+        var results = response.data;
+  
+        for (var i = 0; i < results.length; i++) {
+          var gifDiv = $("<div>");
+  
+          var rating = results[i].rating;
+  
+          var p = $("<p>").text("Rating: " + rating);
+  
+          var animalsImage = $("<img>");
+          animalsImage.attr("src", still);
```

Comment:
Looks like your code is breaking here and it's saying that "still" isn't recognized.